falcon/util/uri.py

```diff
@@ -311,7 +314,7 @@ def parse_query_string(query_string, keep_blank_qs_values=False):
                     # will be the same for 'foo=1,,3' as 'foo=1&foo=&foo=3'.
                     v = [element for element in v if element]
```

Comment:
In fact, i didn't see this case, and for me, it is not a "breaking change" but an error. I think too that handle this case is the thing to do. So i fix it and i add also unit tests for later.

another question, do we accept a mix like that : a=1&a=2&a=3,4 ?
it's a little weird, but the case could exist ... Actually, falcon doesn't handle this case.